Dockerfile

```diff
@@ -8,24 +8,9 @@ RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install
       memcached               \
       postgresql-server       \
       mod_ssl                 \
-      openssh-clients         \
```

Comment:
Is anything here needed for rugged with git repos over ssh?

Reply:
https://github.com/ManageIQ/manageiq-appliance-build/pull/381/commits/eb12c2ad07231559409f58efe0cf318574b419e2 is what was done for appliance

Reply:
If it is, it really should be in the pods image, not here.